src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,194 @@
+ */
+function onLoadMicroscopes(complete) {
+    const microscopesDB = {
+        "Test" : { "hello" : "world" }
+    };
+    complete(microscopesDB);
+}
+
+/* Rough Idea for when on a static or collections page:
+class StaticPageMicrometaWrapper extends React.PureComponent {
+    constructor(props){
+        super(props);
+        this.state = {
+            isSelecting: false
+        };
+        this.completeOnLoad = null;
+    }
+    onLoadMicroscopes(complete){
+        this.setState({ isSelecting: true });
+        this.completeOnLoad = complete;
+    }
+    onReceiveMicroscopeConfig(res){
+        this.completeOnLoad(res);
+    }
+    render(){
+        if (this.state.isSelecting) {
+            return <LinkToSelector/>;
+        }
+}
+*/
+
+function onSaveMicroscope(microscope, complete) {
+    // Do some stuff... show pane for people to browse/select schema.. etc.
+    setTimeout(function() {
+        console.log(microscope);
+        complete(microscope.Name);
+    });
+}
+
+
+/** Path to images directory/CDN. Once is published to NPM, will change to unpkg CDN URL. */
+const imagesPath = "https://raw.githubusercontent.com/WU-BIMAC/4DNMicroscopyMetadataToolReact/master/public/assets/";
+
+let MicroscopyMetadataTool = null;
+
+export class MicroMetaTabView extends React.PureComponent {
+
+    static getTabObject(props){
+        return {
+            'tab' : <span><i className="icon icon-microscope fas icon-fw"/> MicroMeta</span>,
+            'key' : 'micrometa',
+            //'disabled' : !Array.isArray(context.experiments),
+            'content' : (
+                <div className="overflow-hidden">
+                    <h3 className="tab-section-title">
+                        <span>4DN MicroMeta</span>
+                    </h3>
+                    <hr className="tab-section-title-horiz-divider"/>
+                    <MicroMetaTabView {...props} />
+                </div>
+            )
+        };
+    }
+
+    constructor(props){
+        super(props);
+        this.state = {
+            mounted: false
+        };
+    }
+
+    componentDidMount(){
+        const onComplete = () => {
+            this.setState({ mounted: true });
+        };
+
+        if (!MicroscopyMetadataTool) {
+            setTimeout(()=>{
+                // Load in HiGlass libraries as separate JS file due to large size.
+                // @see https://webpack.js.org/api/module-methods/#requireensure
+                require.ensure(['4dn-microscopy-metadata-tool'], (require) => {
+                    MicroscopyMetadataTool = require('4dn-microscopy-metadata-tool').default;
+                    onComplete();
+                }, "microscopy-metadata-bundle");
+            });
+        } else {
+            onComplete();
+        }
+    }
+
+    render(){
+        const { schemas, context, windowWidth, windowHeight } = this.props;
+        const { mounted } = this.state;
+        const tips = object.tipsFromSchema(schemas, context);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['tips' is assigned a value but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)